.rubocop.yml

```
@@ -20,3 +20,7 @@ AllCops:
 
 Style/Documentation:
   Enabled: false
+
+#Metrics/BlockLength:
+#  Exclude:
+#    - '/spec'
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)